lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9058,12 +9058,26 @@ class SwiftifyInfoPrinter {
   }
 
   void printAvailability() {
+    if (!hasMacroParameter("spanAvailability"))
+      return;
     printSeparator();
     out << "spanAvailability: ";
     printAvailabilityOfType("Span");
   }
 
 private:
+  bool hasMacroParameter(StringRef ParamName) {
+    ValueDecl *D = getDecl("_SwiftifyImport");
+    auto *SwiftifyImportDecl = dyn_cast_or_null<MacroDecl>(D);
+    assert(SwiftifyImportDecl);
```

Comment:
Could the compiler ever be used with on old enough SDK that does not have this macro? Or could the compiler be used without the SDK at all? If yes, maybe we don't want to assert here.

Reply:
If that's the case I think we have bigger issues: the macro will still be attached. This only handles whether to use the new or old signature. But it's a good point. @DougGregor what's our stance on using this with an SDK that doesn't contain the macro at all?

Reply:
If that's the case I think we have bigger issues: the macro will still be attached. This only handles whether to use the new or old signature. But it's a good point. @DougGregor what's our stance on using this with an SDK that doesn't contain the macro at all?